src/IBM.WatsonDeveloperCloud/Http/WatsonHttpClient.cs

```diff
@@ -42,6 +42,7 @@ public WatsonHttpClient(string baseUri)
                 this.BaseClient.BaseAddress = new Uri(baseUri);
 
             this.Formatters = new MediaTypeFormatterCollection();
+            this.Formatters.JsonFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("application/octet-stream"));

```

Comment:
@atilatosta @maykonsIBM I was getting a consistent error saying there was a missing formatter for octet-stream. Is there a better place to add this?